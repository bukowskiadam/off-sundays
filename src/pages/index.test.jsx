import React from 'react';
import { shallow } from 'enzyme';

import IndexPage from './index';

describe('index', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<IndexPage />);

        expect(wrapper).toMatchSnapshot();
    });
});
