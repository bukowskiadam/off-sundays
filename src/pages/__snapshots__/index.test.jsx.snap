// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index renders correctly 1`] = `
<div>
  <Component />
  <p>
    Sprawdź, w które niedziele sklepy będą zamknięte:
  </p>
  <ForClientOnly(UpcomingDays, Placeholder />
  <NavLink
    to="/2018/"
  >
    Sprawdź cały rok
  </NavLink>
  <Questions />
  <Footer />
</div>
`;
